core/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -105,9 +106,10 @@ public static void parseMultiLineRequest(RestRequest request, IndicesOptions ind
         String searchType = request.param("search_type");
         String routing = request.param("routing");
 
-        final BytesReference data = request.contentOrSourceParam();
+        final Tuple<XContentType, BytesReference> sourceTuple = request.contentOrSourceParam();
```

Comment:
in msearch, do we support other formats besides json? Maybe we do, I just never realized that.